strongbox-web-core/src/main/java/org/carlspring/strongbox/security/vote/FilterAccessDecisionManager.java

```diff
@@ -1,30 +0,0 @@
-package org.carlspring.strongbox.security.vote;
```

Comment:
I suppose that it related not only with metods, `SB-603` assumes URL access restriction too. This described in project Wiki [here](https://github.com/strongbox/strongbox/wiki/The-strongbox%E2%80%90authentication%E2%80%90users.xml-File#example-of-custom-user-access-model)